{"primaryContentSections":[{"declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"underlying"},{"text":"(","kind":"text"},{"kind":"externalParam","text":"error"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"kind":"text","text":")"}],"languages":["swift"],"platforms":["iOS"]}],"kind":"declarations"}],"kind":"symbol","abstract":[{"text":"An indication of an underlaying network layer error.","type":"text"}],"identifier":{"url":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/underlying(error:)","interfaceLanguage":"swift"},"metadata":{"roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"underlying"},{"text":"(","kind":"text"},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"text":")","kind":"text"}],"role":"symbol","externalID":"s:10Networking12NetworkErrorO10underlyingyACs0C0_p_tcACmF","symbolKind":"case","title":"NetworkError.underlying(error:)","modules":[{"name":"Networking"}]},"hierarchy":{"paths":[["doc:\/\/Networking\/documentation\/Networking","doc:\/\/Networking\/documentation\/Networking\/NetworkError"]]},"schemaVersion":{"major":0,"patch":0,"minor":3},"sections":[],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/networking\/networkerror\/underlying(error:)"]}],"references":{"doc://Networking/documentation/Networking":{"role":"collection","identifier":"doc:\/\/Networking\/documentation\/Networking","abstract":[],"url":"\/documentation\/networking","kind":"symbol","type":"topic","title":"Networking"},"doc://Networking/documentation/Networking/NetworkError":{"title":"NetworkError","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"NetworkError","kind":"identifier"}],"role":"symbol","type":"topic","abstract":[{"text":"An error that occurs during ","type":"text"},{"type":"reference","isActive":true,"identifier":"doc:\/\/Networking\/documentation\/Networking\/Response"},{"type":"text","text":" processing or an underlaying "},{"isActive":true,"type":"reference","identifier":"doc:\/\/Networking\/documentation\/Networking"},{"text":" error.","type":"text"}],"identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError","navigatorTitle":[{"text":"NetworkError","kind":"identifier"}],"kind":"symbol","url":"\/documentation\/networking\/networkerror"},"doc://Networking/documentation/Networking/NetworkError/underlying(error:)":{"fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"underlying","kind":"identifier"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","text":"Error","kind":"typeIdentifier"},{"kind":"text","text":")"}],"abstract":[{"type":"text","text":"An indication of an underlaying network layer error."}],"kind":"symbol","role":"symbol","url":"\/documentation\/networking\/networkerror\/underlying(error:)","type":"topic","title":"NetworkError.underlying(error:)","identifier":"doc:\/\/Networking\/documentation\/Networking\/NetworkError\/underlying(error:)"},"doc://Networking/documentation/Networking/Response":{"url":"\/documentation\/networking\/response","fragments":[{"text":"typealias","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Response","kind":"identifier"}],"role":"symbol","abstract":[{"type":"text","text":"A default "},{"type":"codeVoice","code":"URLSession"},{"type":"text","text":" response tuple."}],"type":"topic","identifier":"doc:\/\/Networking\/documentation\/Networking\/Response","navigatorTitle":[{"kind":"identifier","text":"Response"}],"kind":"symbol","title":"Response"}}}